actions/cloud.tsx

```diff
@@ -89,23 +89,29 @@ export function subscribeCloudSubscription(productId: string) {
     };
 }
 
-export function requestCloudTrial(page: string, email = '') {
+export function requestCloudTrial(page: string, subscriptionId: string, email = ''): ActionFunc {
     trackEvent('api', 'api_request_cloud_trial_license', {from_page: page});
-    return async () => {
+    return async (dispatch: DispatchFunc): Promise<any> => {
         try {
-            await Client4.requestCloudTrial(email);
+            const updatedSubscription = await Client4.requestCloudTrial(subscriptionId, email);
+            if (updatedSubscription) {
+                dispatch({
+                    type: CloudTypes.RECEIVED_CLOUD_SUBSCRIPTION,
+                    data: updatedSubscription,
+                });
+            }
```

Comment:
Also wondering if this is necessary given websockets

Reply:
I think we should keep the one set on the cloud_start_trial_button. Let's clean this one.